### PR TITLE
Communication Guidelines edit

### PR DESCRIPTION
**Pull request by aaronepperson**
This updates the communication guidelines with edits and other updates for style guide formats.

**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

**Comment by mrbobbytables** [comment]
/assign 
to get it on my to-do list

**Comment by parispittman** [comment]
thanks for the PR! I've only done a quick skim and I'm finding a lot of empty [ ]'s after [link descriptions]. examples in the moderation docs. will do a formal review in a bit.

File: communication/mailing-list-guidelines.md
Comment (jberkus) [inline communication/mailing-list-guidelines.md:44]: we should also add a bullet about contacting the CoCC here.

**Comment by mrbobbytables** [comment]
@parispittman 
> thanks for the PR! I've only done a quick skim and I'm finding a lot of empty [ ]'s after [link descriptions]. examples in the moderation docs. will do a formal review in a bit.

The empty [] is another way of representing a reference link and is often put in there by markdown formatters.

Markdown is great for being flexible, but also annoying at how inconsistent it is lol

**Comment by parispittman** [comment]
@mrbobbytables do we have one way or another listed in the style guide? or are we accepting of all ways for markdown?

File: communication/discuss-guidelines.md
Comment (jberkus) [inline communication/discuss-guidelines.md:71]:
```suggestion
our [Code of Conduct Committee](mailto: conduct@kubernetes.io) and describe the situation.
```

**Review by aaronepperson — COMMENTED** [review]
Accepted changes that added COCC bullet point

**Review by markyjackson-taulia — APPROVED** [review]

**Comment by mrbobbytables** [comment]
@parispittman we don't have both ways listed in the style guide, but I'd personally accept both - I've seen both used with the trailing[] frequently injected by linters / auto formatters.

File: communication/moderation.md
Comment (mrbobbytables) [inline communication/moderation.md:13]:
```suggestion
-   Check the [centralized list of administrators][moderators] for contact information.
```

File: communication/moderation.md
Comment (mrbobbytables) [inline communication/moderation.md:20]:
```suggestion
[moderators][] who are responsible for keeping
```

File: communication/moderation.md
Comment (mrbobbytables) [inline communication/moderation.md:99]:
```suggestion
        violation has occurred. Make a judgment call based on our [Code of Conduct][] and Values (see below).
```

links that span multiple lines break Hugo rendering :(

File: communication/moderation.md
Comment (mrbobbytables) [inline communication/moderation.md:212]: This entire block should be updated removing the numbers from the links and be given a friendly name or link in the list directly.

**Review by mrbobbytables — CHANGES_REQUESTED** [review]
Thanks! left some suggestions to markdown changes :)

File: communication/mailing-list-guidelines.md
Comment (jberkus) [inline communication/mailing-list-guidelines.md:96]: @mrbobbytables should we be instructing people to create lists under kubernetes.io now?

Note:  the instructions below this are just reformatting of the instructions already there, so it shouldn't hold up the PR approval.  However, it raises a question for future revisions.

**Review by jberkus — CHANGES_REQUESTED** [review]
Approved if you  make the pending changes.

**Comment by jberkus** [comment]
Please make the cited changes, particularly Bob's fixes to the links.  Then can merge.  Sorry for the long review period, there's a lot of lines in this one!

File: communication/mailing-list-guidelines.md
Comment (mrbobbytables) [reply communication/mailing-list-guidelines.md:96]: I'd do that in a separate commit or separate PR from the style edits, I wouldn't necessarily block on it.

Comment (jberkus) [reply communication/mailing-list-guidelines.md:96]: Right.  More of a flag as a TODO.  Lemme open an issue.

File: communication/mailing-list-guidelines.md
Comment (aaronepperson) [reply communication/mailing-list-guidelines.md:44]: Hey! @jberkus can I confirm what you mean by the CoCC? I'm not sure what that is referring to. Thanks in advance!

**Comment by pymander** [comment]
@jberkus @mrbobbytables I worked with Aaron to do tricky Git stuff and resolve all of the issues we could do, then I did a rebase and force push.

**Comment by jberkus** [comment]
/approved

@parispittman @mrbobbytables can one of you LGTM it?

**Comment by jberkus** [comment]
Grrr.

/approve

File: communication/mailing-list-guidelines.md
Comment (jberkus) [reply communication/mailing-list-guidelines.md:44]: Code of Conduct Committee

Comment (pymander) [reply communication/mailing-list-guidelines.md:44]: @jberkus I addressed this in the latest push!

File: communication/moderation.md
Comment (mrbobbytables) [inline communication/moderation.md:278]: definitely has to be a linter, do you know why its doing that?

**Review by mrbobbytables — CHANGES_REQUESTED** [review]
Thanks!
Just have some suggested link fixes and a few questions regarding the linter/formatter^^;;

File: communication/discuss-guidelines.md
Comment (pymander) [reply communication/discuss-guidelines.md:182]: @mrbobbytables The auto-formatter adds all of those lines. I can certainly remove them!

Comment (jberkus) [reply communication/discuss-guidelines.md:182]: hey, is there any chance you can reformat this?  Otherwise, I'll close it.

**Pull request closed without merging**
